src/braket/aws/aws_quantum_task.py

```diff
@@ -583,10 +578,10 @@ def _(
 ) -> AwsQuantumTask:
     openqasm_program = OpenQASMProgram(
         source=pulse_sequence.to_ir(),
-        inputs=inputs if inputs else {},
+        inputs=inputs or {},
```

Comment:
Again can we make sure `inputs` is never a falsy value

Reply:
I don't quite understand, if inputs is False/None that {} is fine, no?

Reply:
In this case, I think it is fine.